hibernate-rx-api/src/main/java/org/hibernate/rx/RxSession.java

```diff
@@ -11,6 +11,8 @@
 	<T> CompletionStage<Optional<T>> find(Class<T> entityClass, Object id);
 
 	CompletionStage<Void> persist(Object entity);
+	
+	CompletionStage<Void> rxFlush();
```

Comment:
If that's ok I'm going to rename this to just flush for consistency with the other methods and because this is already the RxSession.

Reply:
Oh, right. yes please.